assets/scripts/index.js

```diff
@@ -0,0 +1,126 @@
+$(document).ready(function(){
+
+  document.querySelector(".js-nav-toggle").addEventListener("click",toggleMenuFunction);
+  document.querySelector(".js-top").addEventListener("click",toTop);
+  window.onscroll = function() {scrollFunction()};
+
+  function toggleMenuFunction() {
+    let burger = document.querySelector(".nav__menu-toggle");
+    let burgerMenu = document.querySelector(".nav__menu");
+    let mainnav = document.querySelector(".nav");
+    burger.classList.toggle("open");
+    burgerMenu.classList.toggle("open");
+    mainnav.classList.toggle("open");
+  }
+
+  function toTop() {
+    if(window.scrollTo){
+      window.scrollTo({top:0,behavior:"smooth"})
```

Comment:
@edwartexe we also use a space after commas
```suggestion
      window.scrollTo({top:0, behavior:"smooth"})
```